server/src/main/java/org/opensearch/action/ActionModule.java

```diff
@@ -504,6 +504,7 @@ public ActionModule(
             Stream.of(new RestHeaderDefinition(Task.X_OPAQUE_ID, false))
         ).collect(Collectors.toSet());
         UnaryOperator<RestHandler> restWrapper = null;
+        // Only one plugin is allowed to have a rest wrapper. i.e. Security plugin
```

Comment:
Added a helper comment for other developers to better understand this for loop

Reply:
@cwperks From what I've understood there is no particular order in which plugins are loaded.

And @peternied I'll go ahead and create a PR against main.